sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/api/Azure.Monitor.OpenTelemetry.Exporter.netstandard2.0.cs

```diff
@@ -1,5 +1,9 @@
 namespace Azure.Monitor.OpenTelemetry.Exporter
 {
+    public abstract partial class AzureMonitorBaseExporter<T> : OpenTelemetry.BaseExporter<T> where T : class
```

Comment:
@pakrym what's the common practice for Azure SDKs regarding base class? Would this be a concern in the arch board API review?

Reply:
It seems to be used purely for code sharing. It might get flagged in the review.

Reply:
Thanks @pakrym!

@rajkumar-rangaraj maybe we can reduce the public API surface by making this an internal/private static helper, or using composition if access to member variable is needed.

Reply:
Sure. Considered this approach as lot of new object creation could be avoided and a proper isolation could be maintained between traces/logs/metrics.